agents/plugins/smart

```diff
@@ -82,6 +88,7 @@ if which smartctl > /dev/null 2>&1 ; then
     fi
 
     echo '<<<smart>>>'
+    INPATH_UDEVADM=$(inpath udevadm && echo present)
```

Comment:
Is there a need for `echo present`? If yes, please explain.

Reply:
`inpath`, like you suggested doesn't return anything but an exit code of 0 in case
of success and something else in case it fails. It wouldn't assign anything to `INPATH_UDEVADM`,
because there's no stdout.

One alternative would be to assign `INPATH_UDEVADM=$(command -v udevadm)` directly, like you suggested.
But using a function, assigned somewhere on top of the plugin brings its benefits in case it needs to be changed.